docs/_blog/liana-payjoin.md

```diff
@@ -0,0 +1,45 @@
+
+However, there's a catch: most users do not frequently spend their Bitcoin. A [2024 study](https://www.sciencedirect.com/science/article/pii/S0378437124008045) found that the age of unspent coins follows a power-law distribution (see Figure 3 that study). In practice, this means coins are typically either spent soon after receipt or held for long periods. [Data also suggests](https://charts.bitbo.io/hodl-waves/) that most coins remain unspent for months or years. To avoid unintentionally triggering a recovery path, users often resort to reconciliation of their coins to reset timelocks.
+
+<img alt="Liana time locked coins" src="../assets/liana-coins.png" />
+
+But this reconciliation comes at a cost to privacy. These transactions create an identifiable on-chain footprint, making it easier for on-chain surveillance to estimate a user's balance and transaction history. This is primarily due to the [common input ownership heuristic](https://en.bitcoin.it/wiki/Common-input-ownership_heuristic), which assumes all inputs in a transaction belong to the same entity. Combined with unique transaction patterns that act like fingerprints, these heuristics allow on-chain analysts to cluster a user's addresses and glean sensitive information.
+
+## Payjoin as a Privacy-Preserving Refresh
+
+This is where Payjoin shines. It allows Liana users to refresh near-expiry coins during normal payments. For example, when a user receives a payment via Payjoin, they can opportunistically include expiring coins to refresh their timelocks. This same logic can apply to self-spends&mdash;users can initiate a Payjoin when sending a transaction, embedding refresh into routine activity.
+
+Whether as sender or receiver, users benefit from lower fees, improved privacy, and preservation of their inheritance policies.
+
+<img alt="Liana Payjoin" src="../assets/liana-batch-payment.svg" />
+
+## Caveats
+
+It's worth noting that the common-input heuristic is not defeated when using Payjoin with [P2WSH](https://river.com/learn/terms/p/p2wsh/) inputs&mdash;unless both participants share identical or structurally similar wallet policies. This is because the spending script is revealed on-chain, allowing fingerprinting based on script templates. Liana uses P2WSH by default to maintain compatibility with hardware wallets, which currently have limited support for [Taproot miniscript descriptors aka MiniTapscript](https://github.com/bitcoin/bitcoin/pull/27255).
+
+However, [Taproot inputs](https://thebitcoinmanual.com/articles/pay-to-taproot-p2tr), a newer input type, do not suffer from this issue. With Taproot's key-path spending, the recovery script remains hidden unless explicitly revealed. This makes it possible to conduct Payjoin transactions between users with different wallet policies while breaking the common-input heuristic.
```

Comment:
No however if you're concluding with the connection to taproot already (great transition otherwise)

```suggestion
[Taproot inputs](https://thebitcoinmanual.com/articles/pay-to-taproot-p2tr), which use a newer script type, do not suffer from this issue. With Taproot's key-path spending, the recovery script remains hidden unless it is invoked. Taproot makes it possible to conduct Payjoin transactions between users with different wallet policies and still break the common-input heuristic.
```